src/App.tsx

```diff
@@ -1,98 +1,70 @@
+                        data-cy="perPageSelector"
+                        id="perPageSelector"
+                        className="form-control"
+                        onChange={(event) => { perPageChange(event) }}
+                        value={itemsPerPage}
+                    >
+                        <option value="3">3</option>
+                        <option value="5">5</option>
+                        <option value="10">10</option>
+                        <option value="20">20</option>
+                    </select>
+                </div>
 
-        <label htmlFor="perPageSelector" className="col-form-label col">
-          items per page
-        </label>
-      </div>
+                <label htmlFor="perPageSelector" className="col-form-label col">
+                    items per page
+                </label>
+            </div>
 
-      {/* Move this markup to Pagination */}
-      <ul className="pagination">
-        <li className="page-item disabled">
-          <a
-            data-cy="prevLink"
-            className="page-link"
-            href="#prev"
-            aria-disabled="true"
-          >
-            «
-          </a>
-        </li>
-        <li className="page-item active">
-          <a data-cy="pageLink" className="page-link" href="#1">1</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#2">2</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#3">3</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#4">4</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#5">5</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#6">6</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#7">7</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#8">8</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#9">9</a>
-        </li>
-        <li className="page-item">
-          <a
-            data-cy="nextLink"
-            className="page-link"
-            href="#next"
-            aria-disabled="false"
-          >
-            »
-          </a>
-        </li>
-      </ul>
-      <ul>
-        <li data-cy="item">Item 1</li>
-        <li data-cy="item">Item 2</li>
-        <li data-cy="item">Item 3</li>
-        <li data-cy="item">Item 4</li>
-        <li data-cy="item">Item 5</li>
-      </ul>
-    </div>
-  );
+            <Pagination
+                total={total}
+                perPage={itemsPerPage}
+                currentPage={currentPage}
+                onPageChange={(pageNum) => setCurrentPage(pageNum)}
```

Comment:
```suggestion
                onPageChange={() => setCurrentPage(pageNum)}
```

Reply:
that doesn`t  work : Cannot find name 'pageNum'.
<img width="754" alt="Screenshot 2023-12-05 at 20 14 36" src="https://github.com/mate-academy/react_pagination/assets/132989084/9e362d32-0814-48f1-a5cf-7870164fe920">